programs/aes/crypt_and_hash.c

```diff
@@ -406,6 +406,9 @@ int main(int argc, char *argv[])
          * Check the file size.
          */
         if (mbedtls_cipher_info_get_mode(cipher_info) != MBEDTLS_MODE_GCM &&
+            mbedtls_cipher_info_get_mode(cipher_info) != MBEDTLS_MODE_CTR &&
+            mbedtls_cipher_info_get_mode(cipher_info) != MBEDTLS_MODE_CFB &&
+            mbedtls_cipher_info_get_mode(cipher_info) != MBEDTLS_MODE_OFB &&
```

Comment:
It seems there's quite a bit of duplication here. And while it's test code, might be better to use a local variable for the mode. And while doing that, how about locals for the results of `mbedtls_md_get_size()` and `mbedtls_cipher_get_block_size()`?